Comments/Startup.cs

```diff
@@ -184,18 +184,18 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                     if (reqPath.HasValue && reqPath.Value.Contains("."))
                     {
 						// Map static files paths to the root, for use within the 
-						if (reqPath.Value.Contains("/consultations"))
-						{
-							context.Request.Path = reqPath.Value.Replace("/consultations", "");
-						}
-						else if (reqPath.Value.IndexOf("favicon.ico", StringComparison.OrdinalIgnoreCase) == -1 && reqPath.Value.IndexOf("hot-update", StringComparison.OrdinalIgnoreCase) == -1)
-						{
-							context.Response.StatusCode = 404;
-							throw new FileNotFoundException($"Path {reqPath.Value} could not be found. Did you mean to load '/consultations{context.Request.Path.Value  }' instead?");
-						}
-                    }
-
-                    return next();
+						//if (reqPath.Value.Contains("/consultations"))
```

Comment:
normally i'd want commented out code removed, but i think it's fine in this case for now.